src/javascript/ContentEditor/DesignSystem/ColorPickerInput/ColorPickerInput.jsx

```diff
@@ -36,13 +35,11 @@ export const ColorPickerInput = ({onChange, onBlur, initialValue, readOnly, inpu
     );
 
     return (
-        <div className={styles.root} style={{borderColor: getVisualValue(internalValue)}}>
+        <div ref={containerRef} className={styles.root} style={{borderColor: getVisualValue(internalValue)}}>
             <Input
-                inputRef={htmlInput}
+                className={styles.input}
                 data-sel-readonly={readOnly}
-                variant={{
-                    interactive: InteractiveVariant
-                }}
+                postfixComponents={postfixComponent}
```

Comment:
Can you add `size="big"`? I guess it will be better to have the bigger size for Content Editor.